aspnetcore/host-and-deploy/docker/building-net-docker-images.md

```diff
@@ -0,0 +1,211 @@
+
+* `dotnet/core/aspnet` 
+
+   The sample uses this image for running the app. The image contains the ASP.NET Core runtime and libraries and is optimized for running apps in production. Designed for speed of deployment and app startup, the image is relatively small, so network performance from Docker Registry to Docker host is optimized. Only the binaries and content needed to run an app are copied to the container. The contents are ready to run, enabling the fastest time from `Docker run` to app startup. Dynamic code compilation isn't needed in the Docker model.
+
+## Prerequisites
+
+* [.NET Core 2.2 SDK](https://www.microsoft.com/net/core)
+
+* Docker client 18.03 or later
+
+  * Linux distributions
+     * [CentOS](https://docs.docker.com/install/linux/docker-ce/centos/)
+     * [Debian](https://docs.docker.com/install/linux/docker-ce/debian/)
+     * [Fedora](https://docs.docker.com/install/linux/docker-ce/fedora/)
+     * [Ubuntu](https://docs.docker.com/install/linux/docker-ce/ubuntu/)
+  * [macOS](https://docs.docker.com/docker-for-mac/install/)
+  * [Windows](https://docs.docker.com/docker-for-windows/install/)
+
+* [git](https://git-scm.com/download)
+
+## Download the sample application
+
+* Download the sample by cloning the [.NET Core Docker repository](https://github.com/dotnet/dotnet-docker): 
+
+  ```console
+  git clone https://github.com/dotnet/dotnet-docker
+  ```
+
+## Run the app locally
+
+* Navigate to the project folder at *dotnet-docker/samples/aspnetapp/aspnetapp*.
+
+* Run the following command to build and run the app locally:
+
+  ```console
+  dotnet run
+  ```
+
+* After the application starts, visit `http://localhost:5000` in a browser to verify that the app is working.
+
+* Press Ctrl+C at the command prompt to stop the app.
+
+## Run in a Linux container
+
+* In the Docker client, switch to Linux containers.
+
+* Navigate to the Dockerfile folder at *dotnet-docker/samples/aspnetapp*.
+
+* Run the following commands to build and run the sample in Docker:
+
+  ```console
+  docker build -t aspnetapp .
+  docker run -it --rm -p 5000:80 --name aspnetcore_sample aspnetapp
+  ```
+
+  The `build` command arguments:
+  * Name the image aspnetapp.
+  * Look for the Dockerfile in the current folder (the period at the end).
+
+  The run command arguments:
+  * Allocate a pseudo-TTY and keep it open even if not attached. (Same effect as `--interactive --tty`.)
+  * Automatically remove the container when it exits.
+  * Map port 5000 on the local machine to port 80 in the container.
+  * Name the container aspnetcore_sample.
+  * Specify the aspnetapp image.
+
+* After the application starts, see the home page at `http://localhost:5000` in your web browser.
```

Comment:
I'd make this similar to the previous.
Why are these using HTTP and not HTTPS? Must be a very old tutorial that hasn't been updated.

Reply:
HTTP comes from the original tutorial.  I'll make a separate issue to convert to HTTPS.